### PR TITLE
Include Prisma Client installation and client generation with `migrate dev`

### PR DESCRIPTION
**Pull request by Alwoch**
### Description
- Include information about Prisma Client installation and client generation as a part of what happens when `migrate dev` is run

### CODE REVIEW

**Comment by ruheni** [comment]
Hey 👋

When `prisma migrate dev` is run the first time, Prisma Client is installed and generated under the hood. Did you run into an error? I would be happy to get your feedback running through the quickstart and any challenges you ran into. 🙂

**Review by janpio — CHANGES_REQUESTED** [review]
Because of the auto install of Prisma Client during `generate` we currently prefer not to document this as an additional step.

If the auto install does not work for you, that deserves a bug report. Thanks.

**Comment by Alwoch** [comment]
I appreciate your guidance on this @ruheni @janpio and I've made the suggested changes

**Review by ruheni — DISMISSED** [review]
Thanks for making the updates! 🙏

File: content/100-getting-started/01-quickstart.mdx [hunk 1]
```diff
@@ -113,6 +113,8 @@ Congratulations, you now have your database and tables ready. Let's go and learn
 
 ## 4. Explore how to send queries to your database with Prisma Client
 
+When `npx prisma migrate dev` is run the first time, it installs Prisma Client and runs `prisma generate` to generate a client for your existing models. If you make any new changes to your models, you'll have to rerun `npx prisma generate` to update the client.
```
Comment (ruheni) [inline content/100-getting-started/01-quickstart.mdx]:
```suggestion
When `prisma migrate dev` is run the first time, it installs Prisma Client and runs `prisma generate` to generate a client based on your models. Subsequent changes to your database schema with `prisma migrate dev` will trigger `prisma generate` to regenerate Prisma Client.
```

**Review by ruheni — APPROVED** [review]

**Review by janpio — CHANGES_REQUESTED** [review]
This is at least under the wrong headline.
Generally I am not sure if it fits the flow of the quickstart.

**Comment by Alwoch** [comment]
> This is at least under the wrong headline. Generally I am not sure if it fits the flow of the quickstart.

@janpio I've made some changes, Kindly take a look.

File: content/100-getting-started/01-quickstart.mdx [hunk 2]
```diff
@@ -102,19 +102,18 @@ At this point, you have a Prisma schema but no database yet. Run the following c
 npx prisma migrate dev --name init
 ```
 
-This command did two things:
+This command did three things:
 
 1. It creates a new SQL migration file for this migration in the `prisma/migrations` directory.
-1. It runs the SQL migration file against the database.
+2. It runs the SQL migration file against the database.
+3. Installs Prisma Client and generates a client based on yur models
```
Comment (janpio) [inline content/100-getting-started/01-quickstart.mdx]: This is unfortunately not fully correct. 
`prisma migrate dev` runs `prisma generate` under the hood, which installs Prisma Client and generates a Client.

Comment (Alwoch) [reply content/100-getting-started/01-quickstart.mdx]: I've made the suggested change @janpio Kindly take a look

File: content/100-getting-started/01-quickstart.mdx [hunk 3]
```diff
@@ -102,10 +102,12 @@ At this point, you have a Prisma schema but no database yet. Run the following c
 npx prisma migrate dev --name init
 ```
 
-This command did two things:
+This command did three things:
 
 1. It creates a new SQL migration file for this migration in the `prisma/migrations` directory.
-1. It runs the SQL migration file against the database.
+2. It runs the SQL migration file against the database.
+3. Runs `prisma generate` under the hood, which installs Prisma Client and generates a Client based on your models.
```
Comment (nikolasburk) [inline content/100-getting-started/01-quickstart.mdx]: I actually think this is a useful change but I realise there's an issue with the way this is currently phrased. the initial sentence says "_did_ three things" (past tense) but then the bullets are in present tense. Personally, I'd be happy to merged this if we change the bullets all to past tense 🙏

Comment (Alwoch) [reply content/100-getting-started/01-quickstart.mdx]: Hello @nikolasburk I appreciate this suggestion and have made the recommended change.

File: content/100-getting-started/01-quickstart.mdx [hunk 4]
```diff
@@ -102,10 +102,12 @@ At this point, you have a Prisma schema but no database yet. Run the following c
 npx prisma migrate dev --name init
 ```
 
-This command did two things:
+This command did three things:
+
+1. It created a new SQL migration file for this migration in the `prisma/migrations` directory.
+2. It ran the SQL migration file against the database.
+3. Ran `prisma generate` under the hood, which installed Prisma Client and generated a Client based on your models.
```
Comment (nikolasburk) [inline content/100-getting-started/01-quickstart.mdx]:
```suggestion
3. It ran `prisma generate` under the hood, which installed Prisma Client and generated a Client based on your models.
```

File: content/100-getting-started/01-quickstart.mdx [hunk 5]
```diff
@@ -102,10 +102,12 @@ At this point, you have a Prisma schema but no database yet. Run the following c
 npx prisma migrate dev --name init
 ```
 
-This command did two things:
+This command did three things:
+
+1. It created a new SQL migration file for this migration in the `prisma/migrations` directory.
+2. It ran the SQL migration file against the database.
```
Comment (nikolasburk) [inline content/100-getting-started/01-quickstart.mdx]:
```suggestion
2. It executed the SQL migration file against the database.
```

**Review by janpio — APPROVED** [review]

File: content/100-getting-started/01-quickstart.mdx [hunk 6]
```diff
@@ -102,10 +102,12 @@ At this point, you have a Prisma schema but no database yet. Run the following c
 npx prisma migrate dev --name init
 ```
 
-This command did two things:
+This command did three things:
+
+1. It created a new SQL migration file for this migration in the `prisma/migrations` directory.
+2. It executed the SQL migration file against the database.
+3. It ran `prisma generate` under the hood, which installed Prisma Client and generated a Client based on your models.
```
Comment (nikolasburk) [inline content/100-getting-started/01-quickstart.mdx]:
```suggestion
3. It ran `prisma generate` under the hood (which installed the `@prisma/client` package and generated a tailored Prisma Client API based on your models).
```

**Comment by nikolasburk** [comment]
Hey @Alwoch, sorry this took so long and thank you _so much_ for taking the time to help us improve our docs 💚 that's much appreciated!!

**Pull request merged**